packages/razzle-plugin-scss/README.md

```diff
@@ -16,4 +16,185 @@ Using the plugin with the default options
 module.exports = {
   plugins: ['scss'],
 };
-```
\ No newline at end of file
+```
+
+### With custom options
+
+Please remember that custom options will extends default options using `Object.assign`.
+Array such as postcss.plugins __WILL NOT BE EXTENDED OR CONCATED__, it will override all default plugins.
+
+```js
+// default options
+const defaultOptions = {
```

Comment:
When i show off options. Show an example of how to use with a simple change and shortest possible snippet. Then below show/describe defaults beneath. It's just overwhelming here.

Reply:
Better if I just delete it? I have already shown the default options beneath the example